src/lib/MedicationInformationService.jsx

```diff
@@ -1,3 +1,20 @@
+import codeableConceptUtils from '../model/CodeableConceptUtils.jsx';
+
+// List of medications 
+// TODO: Add more medications
```

Comment:
Is this TODO something that will happen in future tasks?

Reply:
Yes, as it is needed, we can add more medications here.

Reply:
I added some more comments specifying that this is mainly used by the note parser right now